.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -1,6 +1,9 @@
 ---
 name: Feature Request
 about: Suggest a feature
+title: ''
```

Comment:
Note: These changes were done automatically by github, as I created the changes in the github UI as it has a preview :)